example/submit.pbs

```diff
@@ -32,7 +51,7 @@ params[\"minband\"]=36
 params[\"maxband\"]=37
 params[\"minband_soc\"]=36
 params[\"maxband_soc\"]=37
-params[\"nac_method\"]=3
+params[\"nac_method\"]=0
```

Comment:
what about 2?  Actually, I think, we need to prepare 4  folders - with 4 types of calculation setups, so the user could just jump-start such needed calculation type (only keep the pp, md.out files in a single copy, to minimize the size of the repository)